.github/workflows/build.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         - name: cargo-workspaces
           version: '0.2.35'
         - name: cargo-workspaces
-          version: '0.3.4'
+          version: '0.3.6'
```

Comment:
There's something in cargo-workspaces that started failing on newer versions of Rust. It was fixed in `.6`, so I'm just going to upgrade it here.